### PR TITLE
11.0 mig sale force invoiced

### PR DESCRIPTION
**Pull request by andreasdsp**

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: sale_force_invoiced/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+#. Create a sales order and confirm it.
+#. Deliver the products/services.
+#. Create an invoice and reduce the invoiced quantity. The sales order
+   invoicing status is 'To Invoice'.
+#. Lock the Sale Order, to change the status of it to 'Done'.
+#. Check the field 'Force Invoiced'
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/10.0
```
Comment (RoelAdriaans) [inline sale_force_invoiced/README.rst]: Links to old runbot, update to 11.0

File: sale_force_invoiced/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (RoelAdriaans) [inline sale_force_invoiced/__init__.py]: Remove al coding: utf-8 lines, no longer needed with python 3?

**Review by RoelAdriaans — CHANGES_REQUESTED** [review]
Functional test :+1:

**Comment by andreasdsp** [comment]
Thank you for the review. i had remove utf-8 code and change runbot url.

File: sale_force_invoiced/README.rst (hunk 1 above)
Comment (andreasdsp) [reply sale_force_invoiced/README.rst]: done

File: sale_force_invoiced/__init__.py (hunk 2 above)
Comment (andreasdsp) [reply sale_force_invoiced/__init__.py]: done

**Review by RoelAdriaans — APPROVED** [review]

**Review by huyly0909 — APPROVED** [review]

**Review by hieulucky111 — APPROVED** [review]

**Review by novawish — APPROVED** [review]

**Review by DaoTheKien — APPROVED** [review]

**Review by etobella — APPROVED** [review]
LGTM :+1:

**Comment by AaronHForgeFlow** [comment]
ready to merge

**Comment by RoelAdriaans** [comment]
@pedrobaeza I think this can be merged

File: sale_force_invoiced/view/sale_view.xml [hunk 3]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (MiquelRForgeFlow) [inline sale_force_invoiced/view/sale_view.xml]: I think this change should be reverted

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: sale_force_invoiced/view/sale_view.xml (hunk 3 above)
Comment (RoelAdriaans) [reply sale_force_invoiced/view/sale_view.xml]: @mreficent This header is no longer required, assumed is UTF-8. 
See odoo documentation, there is the header is missing: 
https://www.odoo.com/documentation/11.0/reference/data.html
In the newer xml files in default Odoo you can also see that this line is absent.

Comment (MiquelRForgeFlow) [reply sale_force_invoiced/view/sale_view.xml]: But your example is not valid. In https://www.odoo.com/documentation/10.0/reference/data.html the header is also missing.

Comment (pedrobaeza) [reply sale_force_invoiced/view/sale_view.xml]: For being a correct xml with UTF-8 coded words, you should include that attribute and the header. If not, there can be problems depending on the lxml library implementation and version. What is not needed in python 3 is for python files.

Comment (RoelAdriaans) [reply sale_force_invoiced/view/sale_view.xml]: @mreficent I did not say it was new with Odoo 11.. 
In that case Odoo is also doing things wrong, for example in this file: https://github.com/odoo/odoo/blob/11.0/addons/hr_recruitment/views/hr_department_views.xml

(For extra confusion, odoo guidelines specify the utf-8 header for python files, but not for the xml files.. :) )

**Comment by JayVora-SerpentCS** [comment]
Question : When the SO is done, the field becomes editable ! But when the SO is done, in invoice and delivery both are said done, isnt it ?
Sorry if its a confusion.

**Comment by emagdalenaC2i** [comment]
@pedrobaeza I think this PR should be closed because this module is merged in #688

**Pull request closed without merging**